atm.js

```diff
@@ -1,21 +1,123 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//select / create elements to work with
+console.log('ready');
+var checkingBox = $('#check-input');
+var checkingBoxVal = parseInt(checkingBox.val());
+var savingsBox = $('#savings-input');
+var savingsBoxVal = savingsBox.val();
+var checkBalanceDisplay = $('#checking-display');
+var savingsDisplay = $('#savings-display');
+  //checking buttons
+var checkDBtn = $('#checkbtn-d');
+var checkWBtn = $('#checkbtn-w');
+  //savings buttons
+var savingsDBtn = $('#savingsbtn-d');
+var savingsWBtn = $('#savingsbtn-w');
 
-}
 
-function checkingWithdrawal(amount){
+//start w/nothing --> show red
+checkBalanceDisplay.addClass('zero');
+savingsDisplay.addClass('zero');
 
-}
+  //make bank account constructor
+function Accnt() {
+    this.totalBal = 0;
+    this.deposit = function(dep_amnt) {
+      return this.totalBal += dep_amnt;
+    };
+    this.withdrawal = function(withdraw_amnt) {
+      return this.totalBal = this.totalBal - withdraw_amnt;
+    };
+  }
+  //make bank accounts, one for checking, one for savings
+var checkingAccnt = new Accnt();
+var savingsAccnt = new Accnt();
 
-function savingsDeposit(amount){
+//event handlers
+  //checking account
+    //on checking deposit click, deposit
+checkDBtn.click(function() {
+    checkingAccnt.deposit(parseInt(checkingBox.val()));
+    checkingBox.val('');
+    checkBalanceDisplay.removeClass('zero');
+    checkBalanceDisplay.html('$' + checkingAccnt.totalBal);
+    console.log('checking balance total: ' + checkingAccnt.totalBal);
+});
+    //on check withdraw click, withdraw
+checkWBtn.click(function() {
+  var withdraw_amnt = parseInt(checkingBox.val());
+  // Make sure the balance in an account can't go negative. If a user tries to withdraw more money than exists in the account, ignore the transaction.
+    // If a withdrawal can be covered by the balances in both accounts, take the balance of the account withdrawn from down to $0 and take the rest of the withdrawal from the other account.
+  if (withdraw_amnt >= checkingAccnt.totalBal && checkingAccnt.totalBal >= 0 && withdraw_amnt <= savingsAccnt.totalBal) {
+      checkBalanceDisplay.addClass('zero');
+      var checkingLeftOver = checkingAccnt.withdrawal(withdraw_amnt);
+      checkingAccnt.totalBal = 0;
+      checkingBox.val('');
+      checkBalanceDisplay.html('$' + checkingAccnt.totalBal);
```

Comment:
This function is really long, I would suggest abstracting and breaking this up into smaller functions or methods